### ISSUE TITLE
Multiple captures in parallel sometimes fail: (Unix.Unix_error "No such file or directory" unlink)

### ISSUE THREAD

**Issue by hmijail** [facebook/infer#711]
Infer v0.12.0-2dcde3a8

While chasing for another bug (as seen in #710 ), I have found another issue that actually is more frequent. Looks like during parallel builds, infer tries to delete files that are not there.

```
(Unix.Unix_error "No such file or directory" unlink
   "((filename .../infer-out/.start))")

Raised at file "src/core_unix.ml", line 47, characters 4-43
Called from file "integration/Driver.ml" (inlined), line 212, characters 18-35
Called from file "integration/Driver.ml", line 217, characters 45-54
Called from file "integration/Driver.ml", line 542, characters 37-72
Called from file "backend/infer.ml", line 33, characters 2-26
Called from file "backend/infer.ml", line 130, characters 6-52
```

**Comment by hmijail** [comment]
Another example:

```
Uncaught exception:

  (Unix.Unix_error "No such file or directory" unlink
   "((filename .../infer-out/duplicates.txt))")

Raised at file "src/core_unix.ml", line 47, characters 4-43
Called from file "integration/Driver.ml" (inlined), line 202, characters 18-35
Called from file "integration/Driver.ml", line 206, characters 39-48
Called from file "integration/Driver.ml", line 535, characters 40-64
Called from file "backend/infer.ml", line 33, characters 2-26
Called from file "backend/infer.ml", line 130, characters 6-52
```

**Comment by jvillard** [comment]
Would you mind letting me know if this is still happening with the latest version of infer, eg 0.13.0? The incriminated code has changed a bit since this issue and should now be disabled by default anyway.
